infra/email_receipt_inbox/lambdas/parsers/parse_amazon.py

```
@@ -0,0 +1,255 @@
+def find_total(text, labels):
+    """Find 'Label: $x.xx' (same line) in plaintext."""
+    for lab in labels:
+        m = re.search(re.escape(lab) + r"\s*:?\s*" + MONEY, text, re.I)
+        if m:
+            return _decode_amount(m.group(1))
+    return None
+
+
+def find_total_lines(lines, labels):
+    """Find label line, amount on same or one of next 2 lines (HTML layout)."""
+    for i, l in enumerate(lines):
```

Ruff (0.15.21)

[error] Ambiguous variable name: `l`

(E741)